notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1285_top_end_assembly_vibration.ipynb

```diff
@@ -109,9 +109,9 @@
     "# start = Time(\"2024-03-10T18:00:00Z\", scale=\"utc\", format=\"isot\")\n",
```

Comment:
<p>That's a detail, but for me it takes 4.5 min</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/140/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I updated the message to warn about this.